index.js

```diff
@@ -38,9 +38,9 @@ function buildRender(assetManager) {
 	let loader = new nunjucks.FileSystemLoader(".", { noCache: true });
 	let env = new nunjucks.Environment(loader);
 	env.addFilter("assetURL", identifier => assetManager.manifest.get(identifier));
-	return (source, markdown) => {
+	return async (source, markdown) => {
 		if(markdown) {
-			let marked = loadExtension("marked", "failed to load markdown library");
+			let marked = await loadExtension("marked", "failed to load markdown library");
 			marked.setOptions({
 				headerIds: false
 			});
```

Comment:
To save you future troubles: This option was [removed](https://marked.js.org/using_advanced#options), so you probably just need to remove these three lines when upgrading marked.

Reply:
Oh, nice! I will try that 👍

Reply:
Cool, that works :+1: I added a bit of Markdown to the example so it actually tests the thing.